internal/store/persistentvolume.go

```diff
@@ -263,6 +263,27 @@ func persistentVolumeMetricFamilies(allowAnnotationsList, allowLabelsList []stri
 				}
 			}),
 		),
+		*generator.NewFamilyGenerator(
+			"kube_persistentvolume_created",
+			"Unix creation timestamp",
+			metric.Gauge,
+			"",
+			wrapPersistentVolumeFunc(func(p *v1.PersistentVolume) *metric.Family {
+				ms := []*metric.Metric{}
+
+				if !p.CreationTimestamp.IsZero() {
```

Comment:
Can `CreationTimestamp` be nil?

Reply:
I think you're right - looks like it is generated before the object is persisted (https://kubernetes.io/docs/reference/using-api/api-concepts/#generated-values). 

This check exists in multiple places - should I update it everywhere in a separate PR?

Reply:
According to those docs, it should actually be be safe to ignore nils. KSM gets objects only after they are persisted, so the creationTimestamp would be set. It's probably why usage of this field in other metrics works without issues